cpp/include/Ice/OutputStream.h

```diff
@@ -545,6 +551,15 @@ class ICE_API OutputStream : public IceInternal::Buffer
         }
     }
 
+    /**
+     * Writes a byte to the stream.
+     * @param v The byte to write.
+     */
+    void write(std::byte v)
+    {
+        b.push_back(static_cast<std::uint8_t>(v));
```

Comment:
Do you plan to upgrade `b` container to use std::byte too?

Reply:
Yes, for a followup PR after the re-format.